hudi-client/hudi-flink-client/src/main/java/org/apache/hudi/index/state/FlinkInMemoryStateIndex.java

```diff
@@ -132,4 +132,9 @@ public boolean canIndexLogFiles() {
   public boolean isImplicitWithStorage() {
     return false;
   }
+
+  @Override
+  public IndexType indexType() {
+    return IndexType.INMEMORY;
```

Comment:
does this mean, this cannot be used in production as well?  the Flink In Memory state is backed up/restored etc. no?

Reply:
yes